sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1191,37 +1184,24 @@ def task_worker(self, stopping_event, sfp_error_event, y_cable_presence):
 
         helper_logger.log_info("Stop SFP monitoring loop")
 
-    def task_run(self, sfp_error_event, y_cable_presence):
+    def task_run(self, sfp_error_event):
         if self.task_stopping_event.is_set():
             return
 
         self.task_process = multiprocessing.Process(target=self.task_worker, args=(
-            self.task_stopping_event, sfp_error_event, y_cable_presence))
+            self.task_stopping_event, sfp_error_event))
         self.task_process.start()
 
     def task_stop(self):
         self.task_stopping_event.set()
         os.kill(self.task_process.pid, signal.SIGKILL)
 
-    def on_port_config_change(self, stopping_event, y_cable_presence, port_change_event):
+    def on_port_config_change(self, stopping_event, port_change_event):
```

Comment:
@param 'stopping_event' is NOT used - please remove

Reply:
fixed